content/zh-cn/blog/_posts/2023-01-12-protect-mission-critical-pods-priorityclass/index.md

```diff
@@ -0,0 +1,524 @@
+-->
+在根据这些需求创建 PriorityClass 之前，让我们看看 PriorityClass 的基本清单是什么样的，并提出一些先决条件：
+
+```yaml
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: PRIORITYCLASS_NAME
+value: 0 # any integer value between -1000000000 to 1000000000 
+description: >-
+  (Optional) description goes here!
+globalDefault: false # or true. Only one PriorityClass can be the global default.
+```
+
+<!--
+Below are some prerequisites for PriorityClasses:
+-->
+以下是 PriorityClass 的一些先决条件：
+
+<!--
+- The name of a PriorityClass must be a valid DNS subdomain name.
+- When you make your own PriorityClass, the name should not start with `system-`, as those names are reserved by Kubernetes itself (for example, they are used for two built-in PriorityClasses).
+- Its absolute value should be between -1000000000 to 1000000000 (1 billion).
+- Larger numbers are reserved by PriorityClasses such as `system-cluster-critical` (this Pod is critically important to the cluster) and `system-node-critical` (the node critically relies on this Pod). `system-node-critical` is a higher priority than `system-cluster-critical`, because a cluster-critical Pod can only work well if the node where it is running has all its node-level critical requirements met.
+- There are two optional fields:
+  - `globalDefault`: When true, this PriorityClass is used for pods where a `priorityClassName` is not specified. Only one PriorityClass with `globalDefault` set to true can exist in a cluster. If there is no PriorityClass defined with globalDefault set to true, all the pods with no priorityClassName defined will be treated with 0 priority (i.e. the least priority).
+  - `description`: A string with a meaningful value so that people know when to use this PriorityClass.
+-->
+- PriorityClass 的名称必须是有效的 DNS 子域名。
+- 当你创建自己的 PriorityClass 时，名称不应以 `system-` 开头，因为这类名称是被 Kubernetes 本身保留的（例如，它们被用于两个内置的 PriorityClass）。
+- 其绝对值应在 -1000000000 到 1000000000（10 亿）之间。
+- 较大的数值由 PriorityClass 保留，例如 `system-cluster-critical`（此 Pod 对集群至关重要）以及 `system-node-critical`（节点严重依赖此 Pod）。
+  `system-node-critical` 的优先级高于 `system-cluster-critical`，因为集群级别关键 Pod 只有在其运行的节点满足其所有节点级别关键要求时才能正常工作。
+- 额外两个可选字段：
+  - `globalDefault`：当为 true 时，此 PriorityClass 用于未指定 `priorityClassName` 的 Pod。
+    集群中只能存在一个 `globalDefault` 设置为 true 的 PriorityClass。
+    如果没有 PriorityClass 的 globalDefault 设置为 true，则所有未定义 priorityClassName 的 Pod 都将被视为 0 优先级（即最低优先级）。
+  - `description`：具备有意义值的字符串，以便人们知道何时使用此 PriorityClass。
+
+<!--
+{{< note >}}
+Adding a PriorityClass with `globalDefault` set to `true` does not mean it will apply the same to the existing pods that are already running. This will be applicable only to the pods that came into existence after the PriorityClass was created.
+{{< /note >}}
+-->
+{{< note >}}
+添加一个将 `globalDefault` 设置为 `true` 的 PriorityClass 并不意味着它将同样应用于已在运行的现有 Pod。
+这仅适用于创建 PriorityClass 之后出现的 Pod。
+{{< /note >}}
```

Comment:
```suggestion
{{< note >}}
<!--
Adding a PriorityClass with `globalDefault` set to `true` does not mean it will apply the same to the existing pods that are already running. This will be applicable only to the pods that came into existence after the PriorityClass was created.
-->
添加一个将 `globalDefault` 设置为 `true` 的 PriorityClass 并不意味着它将同样应用于已在运行的现有 Pod。
这仅适用于创建 PriorityClass 之后出现的 Pod。
{{< /note >}}
```

Reply:
Fixed in the latest update.